helper/tiup.bash

```diff
@@ -11,6 +11,17 @@ function confirm_str()
 	fi
 }
 
+function enable_opt()
```

Comment:
Maybe use a longer name? like `to_enable_opt` (not a good example)

Reply:
Would `maybe_enable_opt` be better?